_P164_MSswitch.ino

```diff
@@ -0,0 +1,169 @@
+#ifdef USES_P164
+//#######################################################################################################
+//#################################### Plugin 164: MSswitch #############################################
+//#######################################################################################################
+
+// Februray 3, 2019 initial release
+// 433MHZ Transmitters: http://www.rflink.nl/blog2/wiring
+// Known supported auto-learning switches are Nexa, Prove, Telldus & Jula Anslutt
+
+// List of commands:
+// (1) MSswitch,<G,1-16>,<ON/OFF>
+
+// Usage:
+// (1): Turn on switches in group (eg. MSswitch,G,on)
+
+#include <Socketswitch.h>
+
+// Download library from https://github.com/mortensalomon/Socket_Switch
+
+Socketswitch *Plugin_164_msswitch;
+
+#define PLUGIN_164
+#define PLUGIN_ID_164         99
+#define PLUGIN_NAME_164       "MS auto-learning Switch"
+#define PLUGIN_VALUENAME1_164 ""
+
+unsigned long SocketAddress;
```

Comment:
In every location where this value is used, the event variable is present, so you can also directly call `PCONFIG_LONG(0)` or make a define for it for better readability.
For example something like this:
`#define p164_SocketAddress  PCONFIG_LONG(0)`